### PR TITLE
Exclude services in system namespaces when getting service list.

### PR DESCRIPTION
**Pull request by CH3CHO**
### Ⅰ. Describe what this PR did

Exclude services in system namespaces (e.g.: kube-system) when getting service list.

### Ⅱ. Does this pull request fix one issue?

#75

### CODE REVIEW

File: backend/src/main/java/com/alibaba/higress/console/service/ServiceServiceImpl.java [hunk 1]
```diff
@@ -52,11 +52,16 @@ public PaginatedResult<Service> list(CommonPageQuery query) {
                 kubernetesClientService.gatewayServiceEndpoint();
             List<Service> services = new ArrayList<>(registryzServices.size());
             for (RegistryzService registryzService : registryzServices) {
+                String namespace = registryzService.getAttributes().getNamespace();
+
+                if  (kubernetesClientService.isSystemNamespace(namespace)) {
+                    continue;
+                }
```
Comment (johnlanni) [inline backend/src/main/java/com/alibaba/higress/console/service/ServiceServiceImpl.java:59]: 是否再加个isHigressNamespace的判断，higress控制面如果暴露也有隐患

Comment (CH3CHO) [reply backend/src/main/java/com/alibaba/higress/console/service/ServiceServiceImpl.java:59]: 那我直接改一下默认值就行了，这里System Namespace不单指k8s system。或者我把他改名成ProtectedNamespace吧。

Comment (CH3CHO) [reply backend/src/main/java/com/alibaba/higress/console/service/ServiceServiceImpl.java:59]: Done.

**Review by xiangxiuhui — APPROVED** [review]

**Review by xiangxiuhui — APPROVED** [review]

**Review by johnlanni — APPROVED** [review]
LGTM

**Pull request merged**